source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -1185,8 +1196,10 @@ namespace RendererQml
 		{
 			uiOuterRectangle->Property("radius", "3");
 		}
-		uiOuterRectangle->Property("border.color", checkbox.id + ".checked ? '#0075FF' : '767676'");
-		uiOuterRectangle->Property("color", checkbox.id + ".checked ? '#0075FF' : '#ffffff'");
+
+		auto highlightColor = context->GetColor(AdaptiveCards::ForegroundColor::Accent, false, false);
+		uiOuterRectangle->Property("border.color", Formatter() << checkbox.id << ".checked ? " << highlightColor << ": '#b0b0b0'");
+		uiOuterRectangle->Property("color", Formatter() << checkbox.id << ".checked ? " << highlightColor << " : '#ffffff'");
```

Comment:
Will dark mode and light mode color be same?

Reply:
This wasn't mentioned in figma. Will have to discuss this with UX